src/hotspot/share/gc/shenandoah/shenandoah_globals.hpp

```diff
@@ -147,7 +147,7 @@
           "cases. In percents of (soft) max heap size.")                    \
           range(0,100)                                                      \
                                                                             \
-  product(uintx, ShenandoahLearningSteps, 5, EXPERIMENTAL,                  \
+  product(uintx, ShenandoahLearningSteps, 10, EXPERIMENTAL,                 \
```

Comment:
Relatedly, I'd suggest just removing the "(5)" in this comment in `shenandoahGeneration.cpp`:

```
  // Changing the size of the generation will reset the times learned for the heuristic. The heuristic will need to
  // relearn collection performance metrics. This also has the effect of preventing further capacity changes from the
  // heuristics until at least ShenandoahLearningSteps(5) number of cycles has completed.
  void increase_capacity(size_t increment);
  void decrease_capacity(size_t decrement);
```

Reply:
That whole comment is out of date, I'll remove it.